### DISCUSSION TITLE
figma 116.13.3 按照文档中的解决办法尝试了,安装后打开figma 没有插件

### DISCUSSION THREAD

**Discussion by JackGod001** [Moonvy/figmaEX#34] [Q&A]
文件夹权限开了,安装前关闭了figma, 软件名没改,并且将软件放到了应用程序文件夹中.  mac 版本
![image](https://github.com/Moonvy/figmaEX/assets/24069576/55b8e40c-f199-4ad0-80c6-4a5f2091fd7e)

**Comment by yArna** [comment]
安装器没有用了，请使用集成客户端

**Reply by marshloloXX** [reply]
我也是同样的 集成客户端找了好久也没找到在哪里下载🤦

**Reply by marshloloXX** [reply]
请问集成客户端是指什么呀😂。

**Reply by yArna** [reply]
请自行搜索 FigmaEX 下载页
